settings.gradle

```diff
@@ -11,3 +11,4 @@ plugins {
 rootProject.name = 'liquibase-parent'
 
 include 'liquibase'
+include 'bom'
```

Comment:
:x: The name of the project should be `liquibase-bom`, otherwise the publication GAV coordinates will be `io.micronaut.liquibase:micronaut-bom` which would be both confusing (`micronaut-bom`) and not matching what we have in other projects, eg https://github.com/micronaut-projects/micronaut-data/tree/master/data-bom